Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageTags>CSharp MsTest Roslyn CodeAnalysis analyzers Philips</PackageTags>
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <DevelopmentDependency>true</DevelopmentDependency>
-    <Version>1.2.1</Version>
+		<Version>1.2.2</Version>
     <AssemblyVersion>1.0.3.0</AssemblyVersion>
     <FileVersion>1.2.1.0</FileVersion>
```

Comment:
Rev the FileVersion to match.  (Not rev'ving assemblyversion is correct.)

Reply:
Done.